Client/Frontend/Browser/ContextMenuHelper.swift

```diff
@@ -31,9 +32,8 @@ class ContextMenuHelper: NSObject, BrowserHelper, UIGestureRecognizerDelegate {
         browser.webView.configuration.userContentController.addUserScript(userScript)
 
         // Add a gesture recognizer that disables the built-in context menu gesture recognizer.
-        let disableRecognizer = UILongPressGestureRecognizer()
-        disableRecognizer.delegate = self
-        browser.webView.addGestureRecognizer(disableRecognizer)
+        gestureRecognizer.delegate = self
```

Comment:
This creates a reference loop. In the docs, the `delegate` is marked as `unowned(unsafe)` .. is that similar to weak?